pillars/5_HST__POD/ccengine__stage.sls

```diff
@@ -4,3 +4,4 @@ ccengine:
   branch: staging
   cc_cache: false
   fcgid_max_procs: 3
+  translations_update: false
```

Comment:
I assume this is to avoid needing the secrets in staging.  Would there be any advantage to having translations updated here as well?  (For example, to check that they fit in their assigned place in a layout in a nice way or something)